kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/ManageSelectedLessonResource.vue

```diff
@@ -0,0 +1,207 @@
+      @cancel="closeModal"
+      @submit="$router.go(-1)"
+    >
+      {{ closeConfirmationMessage$() }}
+    </KModal>
+  </div>
+
+</template>
+
+
+<script>
+
+  import SidePanelModal from 'kolibri-common/components/SidePanelModal';
+  import { mapState } from 'vuex';
+  import { searchAndFilterStrings } from 'kolibri-common/strings/searchAndFilterStrings';
+  import commonCoreStrings from 'kolibri/uiText/commonCoreStrings';
+  import { ref, getCurrentInstance } from 'vue';
+  import { enhancedQuizManagementStrings } from 'kolibri-common/strings/enhancedQuizManagementStrings';
+  import { PageNames } from '../../../../constants';
+  import commonCoach from '../../../common';
+  import SelectedResources from './SelectedResources';
+
+  export default {
+    name: 'ManageSelectedLessonResources',
+    components: {
+      SidePanelModal,
+      SelectedResources,
+    },
+    mixins: [commonCoach, commonCoreStrings],
+    setup() {
+      const showModal = ref(false);
+      const hasArrayChanged = ref(false);
+      const router = getCurrentInstance().proxy.$router;
+
+      const { saveLessonResources$, numberOfSelectedResource$ } = searchAndFilterStrings;
+      const { closeConfirmationTitle$, closeConfirmationMessage$ } = enhancedQuizManagementStrings;
+
+      function closeModal() {
+        showModal.value = false;
+      }
+
+      function hasModifiedArray() {
+        hasArrayChanged.value = true;
+      }
+
+      function closeSidePanel() {
+        if (hasArrayChanged.value) {
+          showModal.value = true;
+        } else {
+          router.go(-1);
+        }
+      }
+
+      return {
+        saveLessonResources$,
+        numberOfSelectedResource$,
+        closeConfirmationTitle$,
+        closeConfirmationMessage$,
+        showModal,
+        closeModal,
+        closeSidePanel,
+        hasModifiedArray,
+      };
+    },
+    data() {
+      return {
+        resources: [],
+      };
+    },
+    computed: {
+      ...mapState('lessonSummary', ['currentLesson', 'workingResources', 'resourceCache']),
+    },
+    mounted() {
+      setTimeout(() => {
+        this.getResources();
+      }, 2000);
+    },
+    methods: {
+      removeResource(id) {
+        this.resources = this.resources.filter(lesson => lesson.id !== id);
+        this.hasModifiedArray();
+      },
+      navigateToParent(id) {
+        this.$router.push({
+          name: PageNames.LESSONS_ROOT,
+          params: { classId: this.$route.params.classId, lessonId: id },
+        });
+      },
+      getResources() {
+        const response = this.workingResources.map(resource => {
+          const content = this.resourceCache[resource.contentnode_id];
+          if (!content) {
+            return this.missingResourceObj(resource.contentnode_id);
+          }
+          const tally = this.getContentStatusTally(
+            content.content_id,
+            this.getLearnersForLesson(this.currentLesson),
+          );
+
+          const tableRow = {
```

Comment:
This will also be replaced by useResourceselection compasable. We will not need it anymore